app/models/user.rb

```diff
@@ -4,8 +4,9 @@ class User < ApplicationRecord
   has_many :posts, foreign_key: :author_id
   validates :name, presence: true
   validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-
+  enum status: { draft: 0, completed: 1, published: 2 }
   def recent_post
-    Post.where(author: self).order(created_at: :desc).limit(3)
+    # Post.where(author: self).order(created_at: :desc).limit(3)
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much more noisy, It is always a good practice to keep your code clean from such comments. 😉 👍🏻